webapp/src/main/java/org/opentestsystem/rdw/admin/model/GroupSet.java

```diff
@@ -0,0 +1,52 @@
+package org.opentestsystem.rdw.admin.model;
+
+import com.google.common.collect.ImmutableSet;
+
+import java.util.Set;
+
+public class GroupSet {
```

Comment:
ignore: it will be deleted shortly.  I this would have been removed when I did my revert commit.